contrib/opentimelineio_contrib/adapters/cdl.py

```diff
@@ -0,0 +1,70 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright Contributors to the OpenTimelineIO project
+
+"""
+CDL Export Adapter
+This simple adapter allows users to export a collection of .cdl files
+from an OTIO timeline. The ColorCorrection Node ID within the .cdl will use the
+CMX_3600 reel name/Tape of the clip, while the file itself will be named
+using the timeline event name.
+
+To use: otio.adapters.write_to_file(timeline, cdl_output_directory, adapter_name='cdl')
+"""
+import os
+import xml.etree.cElementTree as ET
```

Comment:
Based on https://docs.python.org/3/library/xml.etree.elementtree.html, `cElementTree` is deprecated and from what it says, `xml.etree.ElementTree` will use the fast implementation available.

Reply:
Hi Jean,

Good spot.
I got in a mess after merging main with your linting fixes trying to correct the missing Sign-off's on some commits - decided it was cleanest and easiest to start fresh, so please see new PR here:
https://github.com/AcademySoftwareFoundation/OpenTimelineIO/pull/1376

Changes are identical, but include swapping `cElementTree` for `ElementTree`.

Thanks and apologies for the confusion.